pkgs/development/tools/misc/babeltrace/default.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.efficios.com/babeltrace";
     license = licenses.mit;
     platforms = platforms.linux;
-    maintainers = [ maintainers.bjornfor ];
```

Comment:
Also, there is a common claim among Nixpkgs contributors that it is better to not use `with lib;` for the whole `meta` scope, and use `lib.licenses.`, `lib.platforms.` and `with lib.maintainers;` here - that also could be amended to the `nixfmt` commit of `babeltrace`.